__tests__/input.js

```diff
@@ -88,3 +88,15 @@ test('textarea send on enter', async () => {
 
   expect(base64PNG).toMatchImageSnapshot(imageSnapshotOptions);
 });
+
+test('textarea long text', async () => {
+  const { driver } = await setupWebDriver({ props: { styleOptions }});
+
```

Comment:
Remove empty line.

Reply:
I'm only seeing one empty line here.. Do you want me to remove line 94?

Reply:
No, it looks good. Bad eyes.